src/components/BulkSelect/BulkSelect.js

```diff
@@ -3,11 +3,15 @@ import PropTypes from 'prop-types';
 import { withProps } from 'recompose';
 import theme from '../../theme';
 import styled from 'styled-components';
-import BulkSelectItem from './styles/BulkSelectItem';
+import ToggleButton from '../ToggleButton';
 import BulkSelectBorder from './styles/BulkSelectBorder';
 import BulkSelectDivider from './styles/BulkSelectDivider';
 import BulkSelectItemContainer from './styles/BulkSelectItemContainer';
 
+/**
+ * @deprecated BulkSelect has been deprecated in favor of using [Grid](/#!/Grid) with
+ * [ToggleButton](/#!/ToggleButton), optionally with [Selectable](/#!/Selectable) and/or [DragBox](/#!/DragBox).
+ */
 class BulkSelect extends React.Component {
```

Comment:
Would it make sense to include a debug or warning log or something to indicate that this component has been deprecated? (since it's something that React does)

Reply:
I should have indicated this was an open question rather than a required change.

I was thinking potentially on initial creation of the component in a lifecycle method

Reply:
Okay - I'm a little concerned about spamming logs if I put it into the constructor, so I'm going to defer this for now.